libs/datastructures/include/hpx/tuple.hpp

```diff
@@ -0,0 +1,21 @@
+//  Copyright (c) 2020 ETH Zurich
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/datastructures/tuple.hpp>
+
+namespace hpx {
+    using hpx::util::forward_as_tuple;
+    using hpx::util::get;
+    using hpx::util::ignore;
+    using hpx::util::make_tuple;
+    using hpx::util::tie;
+    using hpx::util::tuple;
+    using hpx::util::tuple_cat;
+    using hpx::util::tuple_element;
+    using hpx::util::tuple_size;
+}    // namespace hpx
```

Comment:
This reminds me of (C++17) `std::apply`, our equivalent is `invoke_fused`. However we would create a clash with our `apply` if we imported it into namespace hpx.

Reply:
Yes, I noticed that while looking at what's in `<tuple>`! There's not much we can do about that without deprecating the current `hpx::apply` though...